packages/data-table-server/src/routes/FetchDataTableData.ts

```diff
@@ -0,0 +1,46 @@
+/**
+ * Tupaia
+ * Copyright (c) 2017 - 2022 Beyond Essential Systems Pty Ltd
+ */
+
+import { Request } from 'express';
+
+import { Route } from '@tupaia/server-boilerplate';
+import { createDataTable } from '../dataTable';
+
+export type FetchDataTableDataRequest = Request<
+  { dataTableCode: string },
+  { data: unknown[] },
+  Record<string, unknown>,
+  Record<string, unknown>
+>;
+
+export class FetchDataTableDataRoute extends Route<FetchDataTableDataRequest> {
```

Comment:
In the data table server we can probably drop DataTable from names, it should be obvious. FetchDataRoute?

Reply:
In my head there was scope for some routes in this server that did things not relating to a specific data-table, but I think it's premature name-spacing. I like `FetchDataRoute` 👍